libraries/botbuilder-dialogs-adaptive/package.json

```diff
@@ -47,7 +47,8 @@
     "node-fetch": "^2.6.7"
   },
   "devDependencies": {
-    "@types/node-fetch": "^2.5.3"
+    "@types/node-fetch": "^2.5.3",
+    "rimraf": "^3.0.2"
```

Comment:
Is it necessary to include rimraf if you are using npx to run the command?

Reply:
Without the _rimraf_ it returns an error that the reference was not found.